src/gui_impl/signal_browser/signal_graphics_item.cpp

```diff
@@ -72,6 +72,7 @@ SignalGraphicsItem::SignalGraphicsItem (QSharedPointer<SignalViewSettings const>
     connect(signal_view_settings.data(), SIGNAL(separatorEnabled(bool)), SLOT(enableSeparator(bool)));
     connect(signal_view_settings.data(), SIGNAL(gridColorChanged(QColor)), SLOT(updateGridColor(QColor)));
     connect(signal_view_settings.data(), SIGNAL(labelColorChanged(QColor)), SLOT(updateLabelColor(QColor)));
+    setToolTip("Ctrl+mouse wheel up/down to scale an indivicual channel");
```

Comment:
After trying this out, I think we should get rid of the tooltip altogether. Once you know this feature you will likely be annoyed by the tooltip.